packages/protocol/lib/web3-utils.ts

```diff
@@ -385,3 +385,25 @@ export function getFunctionSelectorsForContract(contract: any, contractName: str
     })
   return selectors
 }
+
+export async function retryTx(fn: any, args: any[]) {
```

Comment:
Probably rename it to something like `retryTxWithConfirmation` to distinguish it

Reply:
What do you mean by 'distinguish it'? I just moved this from proxy-utils to web3-utils because it seemed to fit better there logically (since it isn't proxy specific) but it's the same function as before.